src/partials/hero.html

```diff
@@ -1,3 +1,28 @@
 <section id="hero" class="section hero-section">
-  <div class="container">Hero</div>
+  <div class="container hero-content">
+      <div class="hero-tittle">
+        <h1>Discover the joy of learning <span class="english-green">English</span> with us!</h1>
+      </div>
+      <button class="btn cta-button">Leave an application</button>
+      
+      <div class="round-button"></div>
+      
+      <div class="desktop-hero-image">
+        <img src="../img/smiley-girl-wearing-headphones-desktop.jpg" 
```

Comment:
wrong, the image is background-image, not <img>, you should put this in the css!
and you should put also the gradient from Figma project, I don't see this